src/Kunstmaan/AdminBundle/Resources/config/console_listener.yml

```diff
@@ -0,0 +1,7 @@
+services:
+    kunstmaan_admin.consolelogger.listener:
+        class: Kunstmaan\AdminBundle\EventListener\ConsoleExceptionListener
```

Comment:
The classname should be configurable via a parameter. This makes it easier to overwrite the listener when needed.